apps/authentication/mixins.py

```diff
@@ -238,7 +238,7 @@ def get_face_code(self):
 
         if not self.is_context_success(context):
             msg = context.get('error_message', '')
-            raise RuntimeError("Face recognition failed,{}".format(msg))
+            raise RuntimeError(msg)
 
         face_code = context.get('face_code')
         if not face_code:
```

Comment:
There are no significant changes to the code at this moment. The current implementation works correctly based on its provided context. However, it is always recommended to check with real-world usage scenarios for full testing before deployment.